src/main/java/org/terasology/navgraph/BitMap.java

```diff
@@ -53,6 +58,12 @@ public boolean isPassable(int x, int y) {
         return isPassable(offset(x, y));
     }
 
+    /**
+     * Function that adds all the possible nodes that you could possibly go to from the current position
+     * @param offset Offset of the current position
+     * @param successors The list into which the successor nodes will be added
+     */
+
```

Comment:
Same here (and for all the following functions where there's a space between the Javadoc and the code itself)